pkg/thing/interactors/request_data.go

```diff
@@ -24,12 +24,12 @@ func (i *ThingInteractor) RequestData(authorization, thingID string, sensorIds [
 		return err
 	}
 
-	if thing.Schema == nil {
```

Comment:
What do you think about adding two new commits for `request_data` and `update_data` adjusts?

Reply:
Done!